### PR TITLE
Bump phpunit/phpunit 10.1.0 and infection/infection to 0.26.20

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

**Review by Ocramius — APPROVED** [review]
Thanks @kukulich!

File: phpunit.xml.dist [hunk 1]
```diff
@@ -11,16 +11,15 @@
...
   </php>
-
   <testsuites>
     <testsuite name="Roave\InfectionStaticAnalysis tests">
       <directory>./test/unit</directory>
     </testsuite>
   </testsuites>
-
-  <coverage>
+  <coverage/>
```
Comment (maks-rafalko) [inline phpunit.xml.dist:19]: IMO it's not needed, as `<source>` is used by default. Or does it bring any other sense here? Would like to know as at Infection we just made an update related to this PHPUnit change, so if it has some meaning here, would like to understand what :)

Comment (kukulich) [reply phpunit.xml.dist:19]: @maks-rafalko I've used PHPUnit `--migrate-configuration`

Comment (maks-rafalko) [reply phpunit.xml.dist:19]: thanks for letting me know, though not clear why it's done by migration script in his way 😃

**Pull request merged**